### DISCUSSION TITLE
How do I obtain ecliptic obliquity in a script?

### DISCUSSION THREAD

**Discussion by kishorekumar62** [Stellarium/stellarium#3231] [Q&A]
I am writing script and would like to obtain the ecliptic obliquity of date. I am unable to find the scripting API call that can give me this value. Can someone help?

Thanks in advance.

**Comment by alex-w** [comment]
It's not possible at the moment

**Reply by alex-w** [reply]
@kishorekumar62 please follow https://github.com/Stellarium/stellarium/issues/3233
